src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,120 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import Class from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { getFilms } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
-
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
-        </div>
+interface Props {
+  addMovie: (movie: Movie) => boolean
+}
+
+export const FindMovie: React.FC<Props> = React.memo(
+  ({ addMovie }) => {
+    const [query, setQuery] = useState<string>('');
```

Comment:
```suggestion
    const [query, setQuery] = useState('');
```
don't add type for primitive values, fix in all places